### PR TITLE
Add animal and event cards

### PR DESCRIPTION
**Pull request by PPrakapavicius**

### CODE REVIEW

**Review by rikius — APPROVED** [review]

File: src/components/animal/AnimalCard.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    if (!!animal?.details?.species && VALID_SPECIES_IDS.includes(animal.details.species.id)) {
+        speciesId = animal.details.species.id;
+    }
+
+    return `${process.env.PUBLIC_URL}/gender${animal.details.gender.id}-species${speciesId}.png`;
+}
+
+const useStyles = makeStyles(() => ({
+    root: {
+        width: 280,
```
Comment (domzic) [inline src/components/animal/AnimalCard.tsx:65]: Why are you setting width if this component is wrapped in grid item with breakpoints `xs={10} md={6} lg={3}` ?

File: src/components/animal/EventCard.tsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+                    <Typography paragraph>{author}</Typography>
+                    <Typography paragraph>{description}</Typography>
+                </CardContent>
+            </Collapse>
+        </Card>
+    );
+}
+
+const useStyles = makeStyles(theme => ({
+    root: {
```
Comment (domzic) [inline src/components/animal/EventCard.tsx:51]: Same here - material ui grid already does that

File: src/components/animal/AnimalCard.tsx [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+                    }
+                />
+            </Card>
+        </Grid>
+    );
+}
+
+function resolveIconSrc(animal: Animal) {
+    if (!animal.details?.gender || !VALID_GENDER_IDS.includes(animal.details.gender.id)) {
+        return `${process.env.PUBLIC_URL}/logo192.png`;
```
Comment (domzic) [inline src/components/animal/AnimalCard.tsx:52]: Use `config.ts`

**Review by domzic — CHANGES_REQUESTED** [review]

File: src/components/animal/EventCard.tsx (hunk 2 above)
Comment (PPrakapavicius) [reply src/components/animal/EventCard.tsx:51]: Will set width to 100% here as now card's width depends on it's content

**Review by domzic — APPROVED** [review]

**Pull request merged**